crates/cheatnet/tests/common/state.rs

```diff
@@ -16,7 +16,7 @@ pub fn create_cached_state() -> CachedState<ExtendedStateReader> {
 }
 
 pub fn create_fork_cached_state(cache_dir: &str) -> CachedState<ExtendedStateReader> {
-    let node_url = "http://188.34.188.184:9545/rpc/v0.5".parse().unwrap();
+    let node_url = "http://188.34.188.184:9545/rpc/v0_6".parse().unwrap();
```

Comment:
While our node supports rpc 0.6.0, atm this is just -rc (latest). It does not have any differences to released 0.6.0, so I just created an issue to update it later on https://github.com/foundry-rs/starknet-foundry/issues/1415. After this issue is done, the slug will probably change to `v0.6`